examples/03_connectivity/plot_signal_extraction.py

```diff
@@ -90,4 +94,53 @@
 plt.subplots_adjust(left=.01, bottom=.3, top=.99, right=.62)
 plt.suptitle('No confounds', size=27)
 
-plt.show()
+###############################################################################
+# Check the relevance of chosen confounds: The distribution of voxel-to-voxel
+# correlations should be tight and approximately centered to zero.
+
+#######################################################################
+# Compute voxel-wise time series with and without confounds removal, using
+# NiftiMasker.
+from nilearn.input_data import NiftiMasker
+brain_masker = NiftiMasker(memory='nilearn_cache', verbose=5)
+voxel_ts_raw = brain_masker.fit_transform(fmri_filename)
+voxel_ts_cleaned1 = brain_masker.fit_transform(fmri_filename,
+                                               confounds=confounds_filename)
+
+# For comparison, compute voxels signals after high variance confounds removal
+from nilearn.image import high_variance_confounds
+hv_confounds = high_variance_confounds(fmri_filename, n_confounds=20)
+voxel_ts_cleaned2 = brain_masker.fit_transform(fmri_filename,
+                                               confounds=hv_confounds)
+
+###############################################################################
+# Compute the voxel-to-voxel Pearson's r correlations
+from sklearn.covariance import EmpiricalCovariance
+from nilearn import connectome
+connectivity_measure = connectome.ConnectivityMeasure(
+    cov_estimator=EmpiricalCovariance(), kind='correlation')
```

Comment:
Why use the empirical covariance rather than the default LW? I am afraid that it is introducing points that are not core to the example.